tests/unit/core/TestInstallAddon.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import { renderIntoDocument } from 'react-addons-test-utils';
 
 import createStore from 'amo/store';
+import { setInstallState } from 'core/actions/installations';
```

Comment:
I prefer to use verb phrasing that reads like English. This helps the reader understand what's happening. In response to your point about helping us identify the type of action better, is that really important? *Set install state* tells you what it does, no? I found `categoriesFetch` awkward to read because I had to unpack it backwards but it didn't seem important enough to complain about when reviewing the patch.